### PR TITLE
fix: support resources that have families without operations

### PR DESCRIPTION
**Pull request by childish-sambino**
OpenAPI fixture updated to verify behavior.

### CODE REVIEW

**Review by beebzz — COMMENTED** [review]
Seeing these local test failures w Java, but they don't reflect on GH actions: 
`/app/twilio-java/src/test/java/com/twilio/TwilioRestTest.java:[50,16] package org.json does not exist
/app/twilio-java/src/test/java/com/twilio/TwilioRestTest.java:[53,19] package org.mockito does not exist
/app/twilio-java/src/test/java/com/twilio/TwilioRestTest.java:[54,19] package org.mockito does not exist
/app/twilio-java/src/test/java/com/twilio/TwilioRestTest.java:[55,19] package org.mockito does not exist
/app/twilio-java/src/test/java/com/twilio/TwilioRestTest.java:[64,26] package org.mockito does not exist
/app/twilio-java/src/test/java/com/twilio/TwilioRestTest.java:[64,1] static import only from classes and interfaces
/app/twilio-java/src/test/java/com/twilio/TwilioRestTest.java:[67,6] cannot find symbol
  symbol:   class Mock
  location: class com.twilio.rest.TwilioRestTest`

File: src/main/java/com/twilio/oai/resource/Resource.java [hunk 1]
```diff
@@ -29,6 +34,50 @@ public Optional<Resource> getParentResource(final IResourceTree resourceTree) {
             .flatMap(parent -> resourceTree.findResource(parent, false));
     }
 
+    public void updateFamily(final OpenAPI openAPI, final IResourceTree resourceTree) {
```
[the remark this answers is not in the document]
Comment (childish-sambino) [reply src/main/java/com/twilio/oai/resource/Resource.java:37]: That's what I had originally, but it's already a method on the _Resource_ class.

**Comment by childish-sambino** [comment]
@beebzz Need to clear Docker cache as it's using an old version of `twilio-java`.

File: examples/java/src/main/java/com/twilio/rest/api/v2010/Call.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+import com.twilio.type.OutboundCallPrice;
+import com.twilio.type.RecordingRule;
+import com.twilio.type.SubscribeRule;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@ToString
+public class Call extends Resource {
+    private static final long serialVersionUID = 145080881849338L;
+
+
```
Comment (beebzz) [inline examples/java/src/main/java/com/twilio/rest/api/v2010/Call.java:77]: Is all this spacing expected?

Comment (childish-sambino) [reply examples/java/src/main/java/com/twilio/rest/api/v2010/Call.java:77]: Yeah, it's just how the templating works with conditional lines.

**Review by beebzz — APPROVED** [review]

**Review by rakatyal — APPROVED** [review]

**Pull request merged**